src/test/java/ScattTest.java

```diff
@@ -24,13 +25,8 @@
     @Test
     public void testSb2Constructor1() {
         String filePath = Utils.getTestResourcePath("WizardSpells.sb2");
-        try {
-            Sb2 wizardSpells = new Sb2(filePath);
-            assertTrue(wizardSpells.getJSONObject() instanceof org.json.JSONObject);
-        } catch (IOException e) {
-            e.printStackTrace();
-            fail("Constructor threw an error");
-        }
+        Sb2 wizardSpells = new Sb2(filePath);
```

Comment:
Yeah, this is a really minimal test.  I'd like to change testFilePathGoodCase to make sure that the correct report is generated.  If testSb2Constructor1 fails then tests like testFilePathGoodCase will also fail.  The only reason to have testSb2Constructor1 is so that we can ruleout the constructor as the source of the problem if our more robust tests start to fail.